one_fm/templates/pages/career_history.js

```diff
@@ -137,13 +137,13 @@ career_history = Class.extend({
   },
   create_company_section_html: function(company_no) {
     var company_section_html = `<div class="row col-lg-12 col-md-12 mb-12 company_${company_no}">
-      <h3>So {{job_applicant.applicant_name}}, Tell me about your ${stringifyNumber(company_no)} Company you worked on?</h3>
+      <h3>So {{job_applicant.applicant_name}}, tell me about the ${stringifyNumber(company_no)} company you worked on?</h3>
```

Comment:
"So {applicant}, tell us about the {company_no} company you worked for!" (no question mark)